coreos-base/oem-ec2-compat/files/grub-ec2.cfg

```diff
@@ -4,7 +4,7 @@ set oem_id="ec2"
 
 # Blacklist the Xen framebuffer module so it doesn't get loaded at boot
 # Disable `ens3` style names, so eth0 is used for both ixgbevf or xen.
-set linux_append="modprobe.blacklist=xen_fbfront net.ifnames=0 transparent_hugepage=never clocksource=tsc tsc=reliable console=tty1 console=ttyS0 elevator=noop"
+set linux_append="modprobe.blacklist=xen_fbfront net.ifnames=0 transparent_hugepage=never clocksource=tsc tsc=reliable console=tty1 console=ttyS0 elevator=noop mount.timeout=300s"
```

Comment:
I was looking at https://www.novell.com/support/kb/doc.php?id=7018491 first. But then I found this https://www.freedesktop.org/software/systemd/man/systemd.mount.html . I think we can also try putting `TimeoutSec=300` for systemd units for devices.